packages/block-library/src/post-comments-link/index.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comments-link` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comments-link` block on the server.
+ *
+ * @param  array    $attributes Block attributes.
+ * @param  string   $content    Block default content.
+ * @param  WP_Block $block      Block instance.
+ * @return string   Returns the rendered link.
+ */
+function render_block_core_post_comments_link( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] )
+		|| isset( $block->context['postId'] )
+		&& ! comments_open( $block->context['postId'] ) ) {
+			return '';
+	}
```

Comment:
Nitpicking, but could we change the alignment of this to make things line-up a bit more, and have `return` nested 1 level less?

```suggestion
	if (
		! isset( $block->context['postId'] ) ||
		isset( $block->context['postId'] ) &&
		! comments_open( $block->context['postId'] )
	) {
		return '';
	}
```